app/helpers/ip_blocker.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
This should be converted to a class in the `lib/munkireport` so that it can be autoloader on request.